app.py

```diff
@@ -90,6 +90,12 @@ def handle_post_experience():
 
     req = request.get_json()
 
+    choice = request.args.get("corrections")
```

Comment:
Hey @galadd 
What happens if the user doesn't send the corrections param?
The get method will return `None` by default if it can't find the key `corrections` you are looking for, so if the user doesn't sends the `corrections` param your condition might broke because you will be trying to compare `None = <string>` which will result in a `TypeError`  because `None` is a special object representing the absence of a value, while a string is a sequence of characters

```
none_value = None
string_value = "Hello"

# Comparing None with a string
result = none_value == string_value  # Raises a TypeError

```

Reply:
If corrections parameter is not passed. The function continues without a return corrections